.travis.yml

```diff
@@ -6,6 +6,7 @@ node_js:
   - "1.8"
   - "2.0"
   - "2.3"
+  - "4.0.0"
```

Comment:
Can this be changed to "4.0" so we can float against the latest patch of 4.0 instead of updating for every patch version?